### ISSUE TITLE
directives using template w/ multi-transclude seem to lose reference to scope in certain cases

### ISSUE THREAD

**Issue by jusopi** [zachsnow/ng-multi-transclude#29]
I use *multi-transclude* to build reusable components.  After updating to **v0.1.4** I notice that the transcluded content that contain reference to outside scopes doesn't bind properly if the multi-transclude directive creates an isolate scope.  

```coffeescript
.directive 'pod', ->
   {
       scope: {}
   }
```
Here is a plunker for it - http://plnkr.co/edit/ASkgPvnlvZlKVX4lipzT?p=preview (in coffeescript btw, let me know if I need to make it in js)

This was not an issue in v0.1.3, be it by error or design.

**Comment by zachsnow** [comment]
I take it this is supposed to be an edit of #28?

**Comment by jusopi** [comment]
yes. sorry, dunno why it duplicated it.  You can delete the other one.

**Comment by gzsombor** [comment]
Any suggestions what could possible go wrong? I have the same problem, in 0.1.3 the components inside the templates are working, but after, it's not.
